code/_helpers/lists.dm

```diff
@@ -110,7 +110,11 @@ proc/listclearnulls(list/list)
 		result = first ^ second
 	return result
 
-//Pretends to pick an element based on its weight but really just seems to pick a random element.
```

Comment:
I wonder what silly person wrote this comment.

There was a minor error in here, the starting value for total, but otherwise it was fine, it's a tried and tested method